packages/webview_flutter/webview_flutter_wkwebview/example/ios/RunnerTests/FWFWebsiteDataStoreHostApiTests.m

```diff
@@ -70,6 +70,8 @@ - (void)testRemoveDataOfTypes {
                                         blockError = error;
                                       }];
   XCTAssertEqualObjects(returnValue, @YES);
+  // Asserts whether the NSNumber value represents a BOOL.
+  XCTAssertEqual(CFBooleanGetTypeID(), CFGetTypeID((__bridge CFTypeRef)(returnValue)));
```

Comment:
Compare to `CFBooleanGetTypeID()`?

Reply:
Oh, I just wasn't reading carefully and thought the right hand side was the expected value and that you were extracting the type from a local variable. I was suggesting using `CFBooleanGetTypeID()` as the expected value instead which is already what you are doing. Oops!

(It is confusing that this test is inconsistent about whether you put the expected value first or second though. The line right above this does the opposite.)

Reply:
Ah, you're right. I switched the order